<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="distLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Choose type of random based on distribution:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="distBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="randomLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
